Buck fails to build!

After a fresh clone of the repo, running `ant` gives these results:
```
$ ant
Buildfile: /home/sean/Coding/buck/build.xml

clear-successful-build:

compile-buck-modules-annotation-processor:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/buck-modules-ap-classes
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/gen
    [javac] Compiling 6 source files to /home/sean/Coding/buck/ant-out/buck-modules-ap-classes
    [javac] [compiled 538 lines in 554 ms: 971.1 lines/s]
    [javac] [6 .class files generated]
      [jar] Building jar: /home/sean/Coding/buck/ant-out/buck-modules-ap.jar

checkversion:

compile-testrunner:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/testrunner/classes
    [javac] Compiling 18 source files to /home/sean/Coding/buck/ant-out/testrunner/classes

compile-android-annotations:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/android-annotations/classes
    [javac] Compiling 6 source files to /home/sean/Coding/buck/ant-out/android-annotations/classes
    [javac] [compiled 251 lines in 412 ms: 609.2 lines/s]
    [javac] [7 .class files generated]

compile-aosp:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/aosp/classes
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/aosp-gen
    [javac] Compiling 64 source files to /home/sean/Coding/buck/ant-out/aosp/classes
    [javac] [compiled 18787 lines in 984 ms: 19092.4 lines/s]
    [javac] [146 .class files generated]

compile-manual-gen-src:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/src-gen/classes
    [javac] Compiling 109 source files to /home/sean/Coding/buck/ant-out/src-gen/classes

tracing-api:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/tracing-api/classes
    [javac] Compiling 2 source files to /home/sean/Coding/buck/ant-out/tracing-api/classes
    [javac] [compiled 182 lines in 308 ms: 590.9 lines/s]
    [javac] [3 .class files generated]
      [jar] Building jar: /home/sean/Coding/buck/ant-out/tracing-api/tracing-api.jar

bootstrapper:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/bootstrapper/classes
    [javac] Compiling 26 source files to /home/sean/Coding/buck/ant-out/bootstrapper/classes
    [javac] [compiled 1665 lines in 482 ms: 3454.3 lines/s]
    [javac] [28 .class files generated]
      [jar] Building jar: /home/sean/Coding/buck/ant-out/bootstrapper/bootstrapper.jar

buckfilesystem:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/buckfilesystem/classes
    [javac] Compiling 11 source files to /home/sean/Coding/buck/ant-out/buckfilesystem/classes
    [javac] [compiled 1548 lines in 495 ms: 3127.2 lines/s]
    [javac] [15 .class files generated]
      [jar] Building jar: /home/sean/Coding/buck/ant-out/buckfilesystem/buckfilesystem.jar

package-resources:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/classes/com/facebook/buck/android
     [copy] Copying 1 file to /home/sean/Coding/buck/ant-out/classes/com/facebook/buck/android
     [copy] Copying 1 file to /home/sean/Coding/buck/ant-out/classes/com/facebook/buck/android
     [copy] Copying 1 file to /home/sean/Coding/buck/ant-out/classes/com/facebook/buck/android
     [copy] Copying 1 file to /home/sean/Coding/buck/ant-out/classes/com/facebook/buck/android

dx-check-build:

dx:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/dx_classes
    [javac] Compiling 385 source files to /home/sean/Coding/buck/ant-out/dx_classes
    [javac] /home/sean/Coding/buck/third-party/java/dx/src/com/android/dx/merge/DexMerger.java:269: warning: [deprecation] Integer(int) in Integer has been deprecated
    [javac]                 l.add(new Integer(dex));
    [javac]                       ^
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 1 warning
    [javac] Creating empty /home/sean/Coding/buck/ant-out/dx_classes/com/android/dx/rop/package-info.class
    [javac] Creating empty /home/sean/Coding/buck/ant-out/dx_classes/com/android/dx/ssa/package-info.class
      [jar] Building jar: /home/sean/Coding/buck/ant-out/dx.jar

compile-kotlin:
    [mkdir] Created dir: /home/sean/Coding/buck/ant-out/kotlin/kotlin_classes
  [kotlinc] Compiling [/home/sean/Coding/buck/src/com/facebook/buck/jvm/java/abi/kotlin] => [/home/sean/Coding/buck/ant-out/kotlin/kotlin_classes]

compile:
    [javac] Compiling 3128 source files to /home/sean/Coding/buck/ant-out/classes
    [javac] [compiled 603881 lines in 11386 ms: 53037.1 lines/s]
    [javac] [5720 .class files generated]
    [javac] ----------
    [javac] 1. ERROR in /home/sean/Coding/buck/src/com/facebook/buck/jvm/java/abi/JarReader.java (at line 71)
    [javac]     fileSystem = FileSystems.newFileSystem(jarPath, null);
    [javac]                              ^^^^^^^^^^^^^
    [javac] The method newFileSystem(Path, ClassLoader) is ambiguous for the type FileSystems
    [javac] ----------
    [javac] ----------
    [javac] 2. ERROR in /home/sean/Coding/buck/src/com/facebook/buck/jvm/java/abi/java11/SourceVersionUtils.java (at line 27)
    [javac]     switch (version) {
    [javac]             ^^^^^^^
    [javac] The enum constant RELEASE_12 should have a corresponding case label in this enum switch on SourceVersion. To suppress this problem, add a comment //$CASES-OMITTED$ on the line above the 'default:'
    [javac] ----------
    [javac] 3. ERROR in /home/sean/Coding/buck/src/com/facebook/buck/jvm/java/abi/java11/SourceVersionUtils.java (at line 27)
    [javac]     switch (version) {
    [javac]             ^^^^^^^
    [javac] The enum constant RELEASE_13 should have a corresponding case label in this enum switch on SourceVersion. To suppress this problem, add a comment //$CASES-OMITTED$ on the line above the 'default:'
    [javac] ----------
    [javac] ----------
    [javac] 4. ERROR in /home/sean/Coding/buck/src/com/facebook/buck/util/unarchive/Unzip.java (at line 320)
    [javac]     try (FileSystem zipFs = FileSystems.newFileSystem(archiveAbsolutePath, null)) {
    [javac]                                         ^^^^^^^^^^^^^
    [javac] The method newFileSystem(Path, ClassLoader) is ambiguous for the type FileSystems
    [javac] ----------
    [javac] 4 problems (4 errors)

BUILD FAILED
/home/sean/Coding/buck/build.xml:592: The following error occurred while executing this line:
/home/sean/Coding/buck/build.xml:534: Compile failed; see the compiler error output for details.

Total time: 23 seconds
```

--- a/README.md
+++ b/README.md
@@ -17,6 +17,8 @@ Since Buck is used to build Buck, the initial build process involves 2 phases:
     cd buck
     ant
 
+You must be using Java 8 or 11 for this to compile successfully. If you see compilation errors from ant, check your `JAVA_HOME` is pointing at one of these versions.
+
 ##### 2. Use bootstrapped version of Buck to build Buck:
 
     ./bin/buck build --show-output buck

